app/assets/javascripts/discourse/app/components/topic-navigation-popup.js

```diff
@@ -1,12 +1,32 @@
 import Component from "@ember/component";
 import { action } from "@ember/object";
+import discourseComputed from "discourse-common/utils/decorators";
 
 export default Component.extend({
   tagName: "",
   hidden: false,
 
```

Comment:
should probably set noticeId or noticeKey to null, not sure which one is used

Reply:
noticeKey is a computed property. noticeId is a received attribute. I do not think any of them has to be defined here, right?

Reply:
noticeId should be